specification.markdown

```diff
@@ -60,6 +60,8 @@ This document defines specific terms in order to define the format and applicati
 
 `Descendant Element`: A `Descendant Element` is a relative term to describe any `EBML Elements` contained within a `Master Element`, including any of the `Child Elements` of its `Child Elements`, and so on.
 
+`Void Element`: A `Void Element` is an element used to void damaged data or reserve space in a sub-element for later use.
```

Comment:
Maybe better to avoid the `void` in the definition, perhaps overwrite? Also element->`Element`. The term sub-element may be misused. I'd think this is like a `Child Element` but that doesn't make sense. Maybe you mean within a `Master Element`. Such as `<Segment><Void/><Tracks/><Clusters/></Segment>`.